src/common/state/View2DAttributes.code

```diff
@@ -1,15 +1,60 @@
-Target: xml2java
-Constant: DEFAULT_FULL_FRAME_AUTO_THRESHOLD
-Declaration: DEFAULT_FULL_FRAME_AUTO_THRESHOLD
-Definition: 
-    public final static double DEFAULT_FULL_FRAME_AUTO_THRESHOLD = 100.0;
+Target: xml2python
+Code: PyView2DAttributes.C
+Prefix:
+//
+// The type description structure
+//
+static PyNumberMethods View2DAttributes_as_number = {
+    (binaryfunc)View2DAttributes_Add, /*nb_add*/
+    (binaryfunc)0, /*nb_subtract*/
+    (binaryfunc)View2DAttributes_Mul, /*nb_multiply*/
+    (binaryfunc)0, /*nb_divide*/
+    (binaryfunc)0,    /*nb_remainder*/
+    (binaryfunc)0,    /*nb_divmod*/
+    (ternaryfunc)0,    /*nb_power*/
+    (unaryfunc)0,    /*nb_negative*/
+    (unaryfunc)0,    /*nb_positive*/
+    (unaryfunc)0,    /*nb_absolute*/
+    (inquiry)0,    /*nb_nonzero*/
+    (unaryfunc)0,    /*nb_invert*/
+    (binaryfunc)0,    /*nb_lshift*/
+    (binaryfunc)0,    /*nb_rshift*/
+    (binaryfunc)0,    /*nb_and*/
+    (binaryfunc)0,    /*nb_xor*/
+    (binaryfunc)0,    /*nb_or*/
+    0,            /*nb_coerce*/
+    (unaryfunc)0,    /*nb_int*/
+    (unaryfunc)0,    /*nb_long*/
+    (unaryfunc)0,    /*nb_float*/
+    (unaryfunc)0,    /*nb_oct*/
+    (unaryfunc)0,     /*nb_hex*/
+    0,            /*nb_inplace_add*/
+    0,            /*nb_inplace_subtract*/
+    0,            /*nb_inplace_multiply*/
+    0,            /*nb_inplace_divide*/
+    0,            /*nb_inplace_remainder*/
+    0,            /*nb_inplace_power*/
+    0,            /*nb_inplace_lshift*/
+    0,            /*nb_inplace_rshift*/
+    0,            /*nb_inplace_and*/
+    0,            /*nb_inplace_xor*/
+    0,            /*nb_inplace_or*/
+}; 
+Postfix:
 
 Target: xml2atts
 Constant: DEFAULT_FULL_FRAME_AUTO_THRESHOLD
 Declaration: static const double DEFAULT_FULL_FRAME_AUTO_THRESHOLD;
 Definition: const double View2DAttributes::DEFAULT_FULL_FRAME_AUTO_THRESHOLD = 100.0;
 
 
+Target: xml2java
+Constant: DEFAULT_FULL_FRAME_AUTO_THRESHOLD
+Declaration: DEFAULT_FULL_FRAME_AUTO_THRESHOLD
+Definition:     public final static double DEFAULT_FULL_FRAME_AUTO_THRESHOLD = 100.0;
```

Comment:
The output in the generated java code isn't indented properly. I suspect if you put the "public ..." on the next line indented 4 spaces that it may indent it properly.